src/graphql/admin/index.ts

```diff
@@ -18,5 +16,4 @@ if (isDev) {
 export const gqlAdminSchema = new GraphQLSchema({
   query: QueryType,
   mutation: MutationType,
-  types: ALL_INTERFACE_TYPES,
```

Comment:
This is needed to include interface types in the schema

Reply:
even for admin when those types are not used AFAIK?